src/main/java/org/omnifaces/util/Components.java

```diff
@@ -265,6 +276,24 @@ public static boolean isRendered(UIComponent component) {
         return true;
     }
 
+    /**
+     * Disable the passed UI component.
+     * @param component The component to disable.
+     * @since 4.5
+     */
+    public static void disableInputComponent(UIComponent component) {
```

Comment:
On second thought:
- `UIComponent component` argument should be `UIInput input`
- method name can be shortened to `disableInput`
- the methods should be moved into section starting with comment `// Form`, probably after existing `resetInputs()` method